docs/release-notes/FSharp.Compiler.Service.md

```diff
@@ -0,0 +1,21 @@
+---
+category: Release Notes
+categoryindex: 600
+index: 4
+title: FSharp.Compiler.Service
+---
+
+# FSharp.Compiler.Service
+
+## Unreleased
+
+### Fixes
+
+* Miscellaneous fixes to parentheses analysis. ([PR #16262](https://github.com/dotnet/fsharp/pull/16262))
+* Correctly handle assembly imports with public key token of 0 length. ([Issue #16359](https://github.com/dotnet/fsharp/issues/16359), [PR #16363](https://github.com/dotnet/fsharp/pull/16363))
+
+## 43.8.100 - 2023-11-14
```

Comment:
Who, when and how maintains such a header?

Overall I like the "sketch" of this in terms of how it looks and feels, but I am missing a process for making sure it is correct and correctly mappable.